src/excmds.ts

```diff
@@ -1080,6 +1079,25 @@ export async function buffer(index: number | '#') {
 
 // {{{ SETTINGS
 
+/** Equivalent to vim's `:command`. Maps one ex-mode command to another.
+ * 
+ * Examples:
+ *  - `command t tabopen`
+ *  - `command tn tabnext_gt`
+ * 
+ * Note that this is only for excmd->excmd mappings. To map a normal-mode
+ * command to an excommand, see [[bind]].
+ * 
+ * See also:
+ *  - [[bind]]
+ *  - [[reset]]
+ */
+//#background
+export function command(name: string, ...definition: string[]) {
+    const def = definition.join(" ")
+    config.set("exaliases", name, def)
```

Comment:
This should be `config.set("exaliases", def, name)` - config.set has a slightly strange argument order so that the number of arguments given to it can change (and JS doesn't have multiple dispatch).